### PR TITLE
[zh-cn] sync device-plugins service feature-gates

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -735,7 +735,7 @@ ensure your kubelet services are started with the following flags:
 -->
 ## 设备插件与拓扑管理器的集成   {#device-plugin-integration-with-the-topology-manager}
 
-{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:738]: state is "stable"

**Comment by tengqm** [comment]
Thanks.
/approve

**Comment by tanjunchen** [comment]
Thanks. /lgtm

**Pull request merged**